File: specification/confluent/resource-manager/Microsoft.Confluent/preview/2025-08-18-preview/examples-js/Organization_CreateAPIKey_MaximumSet_Gen.js
const { ConfluentManagementClient } = require("@azure/arm-confluent");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to creates API key for a schema registry Cluster ID or Kafka Cluster ID under a environment
 *
 * @summary creates API key for a schema registry Cluster ID or Kafka Cluster ID under a environment
 * x-ms-original-file: 2025-08-18-preview/Organization_CreateAPIKey_MaximumSet_Gen.json
 */
async function organizationCreateAPIKeyMaximumSet() {
  const credential = new DefaultAzureCredential();
  const subscriptionId = "DC34558A-05D3-4370-AED8-75E60B381F94";
  const client = new ConfluentManagementClient(credential, subscriptionId);
  const result = await client.organization.createAPIKey(
    "rgconfluent",
    "pzvuoywx",
    "jqoxoahobqmhnklw",
    "ypyzlfhbml",
    { name: "izlvofweryqgdgq", description: "vdxsmrddjlsfcsnwjezjraxgbkn" },
  );
  console.log(result);
}

File: specification/confluent/resource-manager/Microsoft.Confluent/preview/2025-08-18-preview/examples-java/Organization_CreateAPIKey_MaximumSet_Gen.java

import com.azure.resourcemanager.confluent.models.CreateApiKeyModel;

/**
 * Samples for Organization CreateApiKey.
 */
public final class Main {
    /*
     * x-ms-original-file: 2025-08-18-preview/Organization_CreateAPIKey_MaximumSet_Gen.json
     */
    /**
     * Sample code: Organization_CreateAPIKey_MaximumSet.
     * 
     * @param manager Entry point to ConfluentManager.
     */
    public static void
        organizationCreateAPIKeyMaximumSet(com.azure.resourcemanager.confluent.ConfluentManager manager) {
        manager.organizations().createApiKeyWithResponse("rgconfluent", "pzvuoywx", "jqoxoahobqmhnklw", "ypyzlfhbml",
            new CreateApiKeyModel().withName("izlvofweryqgdgq").withDescription("vdxsmrddjlsfcsnwjezjraxgbkn"),
            com.azure.core.util.Context.NONE);
    }
}
